src/pages/settings/InitialPage.js

```diff
@@ -113,10 +120,17 @@ const defaultMenuItems = [
 const InitialSettingsPage = ({
     myPersonalDetails,
     network,
+    numberFormat,
     session,
     policies,
     translate,
+    userWallet,
 }) => {
+    const walletBalance = numberFormat(
+        userWallet.availableBalance,
```

Comment:
It looks like if the user has no availableBalance, this can be `$NaN`, so I think we want a fallback of 0 here.

<img src='https://user-images.githubusercontent.com/3981102/127245313-ca74cde9-7bc8-4a2e-b499-adc3b5c75ea4.png' width='300'/>